### PR TITLE
Add Design to the design think build process

### PR DESCRIPTION
**Pull request by lukefrake**
Also done some work on anchors
Put in place holders for case studies
Tidied up a few accessibility issues

### CODE REVIEW

**Comment by f000** [comment]
good work! btw I think we need to improve semantics and document outlines

File: _process/design.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+---
+
+<div class="banner banner-large" style="background-image:url(/assets/images/design.jpg)"></div>
+
+<div class="row">
+
+    <article class="column medium-7 center">
+        <h1>Design</h1>
+
+        <p>At <a href="/" action="Click here to go back to the home page">Space Between</a>, we find the most valuable way to kick off a project is by running a design sprint. This is the part digital development process that a lot of agencies miss off, but it's an absolutely necessity for delivering you a successful digital solution.</p>
```
Comment (MCooke) [inline _process/design.html]: part of a digital

File: _process/design.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            </cite>
+        </blockquote>
+
+        <p>We break sprints down into the following 6 stages:</p>
+
+        <h3><i class="fa fa-question"></i> Prepare</h3>
+
+        <p>The key to the success of this process is getting the right people in the room. This should be a member of each team at your company, if you have engineers, we should get one in, if you've got sales reps, bring one of them as well. Everybody will have knowledge on an area, and you won't be able to get that out of them if they're not around. So firstly make sure you clear everybody's calendars.</p>
+
+        <p>Theres a bunch of other parts to prep for, but we'll help out with that when we get to it.</p>
```
Comment (MCooke) [inline _process/design.html]: There's

File: _process/design.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+        <h3><i class="fa fa-question"></i> Download</h3>
+
+        <p>As we've said before, everybody in the room has different experiences and knowledge, all of which might, and probably will be, very be helpful.</p>
+
+        <p>We've got a few exercises we find really helpful when getting people to share this information, from laying out what you and your competitors currently have, to team interviews and everything in between. We keep these discussions short (maximum of 10 minutes) this keeps everything moving while making sure everybody is interested and happy.</p>
+
+        <p>We start at the beginning, meaning that even if people in the room are familiar with the concept, it gets people thinking as a beginning, which the vast majority of your users will be.</p>
```
Comment (MCooke) [inline _process/design.html]: as a beginner

File: _process/design.html [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+        <p>Now we've got everybody thinking as a beginner, it's much easier to encouraging everybody to have a &#8220;How might we&hellip;&#8221; approach, these together ensure we get some juicy ideas.</p>
```
Comment (MCooke) [inline _process/design.html]: encourage

File: _process/design.html [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+        <h3><i class="fa fa-pencil"></i> Sketch</h3>
+
+        <p>Everybody loves the Sketch phase, although most think they won't.</p>
+
+        <p>In Sketch day we try and keep everybody separated, this allows people to go into maximum detail but minimising the amount of design by committee, which at this stage isn't what we want.</p>
```
Comment (MCooke) [inline _process/design.html]: On Sketch day

File: _process/design.html [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+        <p>Again we come with tricks up our sleeves here, and we use 4 different techniques here to help people, they are Notes, Mind Map, Crazy 8s, and Storyboarding.</p>
```
Comment (MCooke) [inline _process/design.html]: use of 'here' twice

they are; Notes

File: _process/design.html [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+
+        <p>After this is completed we critique and use weighted voting to find the best ideas from the bunch.</p>
+
+        <h3><i class="fa fa-thumbs-up"></i> Decision time</h3>
```
Comment (MCooke) [inline _process/design.html]: Decision Time

File: _process/design.html [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+
+        <p>At this point in the process we'll have ideas flowing around, which is exactly what we want, but that does come with some difficulties. We now have to decide what ones to take into the prototype phase.</p>
+
+        <p>We'll spend today finalising what we are going to make in our prototype session, working out the storyboard and planning.</p>
+
+        <h3><i class="fa fa-paint-brush"></i> Prototype</h3>
+
+        <p>We have tips and tricks to make this day work perfectly. We split the room into teams, this means everybody has their own area to look after.</p>
+
+        <p>Nothing is gets coded at this point, our prototypes can be built in Keynote, PowerPoint, Photoshop etc&hellip; but we can help with all of that at the time.</p>
```
Comment (MCooke) [inline _process/design.html]: Nothing gets coded

File: _process/design.html [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+
+        <p>At the end of this section, we'll have had a great looking and workable prototype, which we can show to your users.</p>
```
Comment (MCooke) [inline _process/design.html:64]: we'll have a great

**Pull request merged**